background.ts

```diff
@@ -91,9 +91,10 @@ mainEvents.on('settings-update', async(newSettings) => {
     setLogLevel('info');
   }
   k8smanager.debug = newSettings.debug;
-  const newPathManager = getPathManagerFor(newSettings.pathManagementStrategy);
 
-  if (newPathManager.strategy !== newSettings.pathManagementStrategy) {
+  if (pathManager.strategy !== newSettings.pathManagementStrategy) {
+    const newPathManager = getPathManagerFor(newSettings.pathManagementStrategy);
+
     await pathManager.remove();
     pathManager = newPathManager;
```

Comment:
I think we could remove the above
```
const newPathManager = getPathManagerFor(newSettings.pathManagementStrategy);
```
and replace this line with
```
pathManager = getPathManagerFor(newSettings.pathManagementStrategy);
```

Reply:
Very nice, thanks 🙂